docs/_templates/changelog/v3.0.0.md

```diff
@@ -0,0 +1,25 @@
+## 📦 [Bookshelf - 3.0.0](https://github.com/mcbookshelf/Bookshelf/releases/tag/v3.0.0)
+
+> **🚂 The Pipeline Update!**
+>
+> *This release overhauls the project structure with Beet, focusing on modularity and isolation of each module. It also introduces a new version loader that resolves conflicts between multiple versions.*
+>
+> **⚠️ BREAKING CHANGES**: This new version has a few breaking changes!
+
+- ⚠️ **[#281](https://github.com/mcbookshelf/Bookshelf/issues/281)** - Merged the `bs.biome` and `bs.weather` modules into a single `bs.environment` module.
+- 🎉 **[#272](https://github.com/mcbookshelf/Bookshelf/issues/272)** - Added a new automatically generated `bs.load` module, eliminating the need to manually create `load` and `unload` tags for each module. Bookshelf now also uses **LanternLoad** to load its modules.
+- ⚡ **[#285](https://github.com/mcbookshelf/Bookshelf/pull/285)** - Simplified testing by automating batch specification and removing the need for a `__boot__` function.
```

Comment:
Is it a breaking change?

Reply:
Breaking changes have the ⚠️ emoji. I've tried to follow this template:
- ⚠️ **[#id](issue/pr)** - Breaking Changes
- 🎉 **[#id](issue/pr)** - New Modules
- ✨ **[#id](issue/pr)** - New Features
- ⚡ **[#id](issue/pr)** - Enhancements
- 🐛 **[#id](issue/pr)** - Bug Fixes
- 📝 **[#id](issue/pr)** - Documentation
- ⚙️ **[#id](issue/pr)** - GitHub & CI/CD